packages/rollup/src/rollup_bundle.bzl

```diff
@@ -304,6 +318,16 @@ def _rollup_bundle(ctx):
     # See CLI documentation at https://rollupjs.org/guide/en/#command-line-reference
     args = ctx.actions.args()
 
+    if ctx.attr.supports_workers:
+        # Set to use a multiline param-file for worker mode
+        args.use_param_file("@%s", use_always = True)
+        args.set_param_file_format("multiline")
+
+    # TODO: enforce here? Currently in js to allow a subset of args...
```

Comment:
I decided to let the .js fail on unknown args instead of disallowing any args like here. WDYT?

This allows supporting "key parameters" which aren't actually required, but might be worth supporting? I mainly switched from enforcing here to in .js to support `--environment` which one of the unit tests uses...